### ISSUE TITLE
Error installation script and APK no work

### ISSUE THREAD

**Issue by bobstar-fr** [f4bpp/HotSpot-Remote#1]
Hello
Equipment condition : spotnikv4.2 (RJ45, RI49 previously installed) , Samsung S10e (Android13) with WiFi on same router
 
1) The installation script (curl -fsSL https://raw.githubusercontent.com/f4bpp/HotSpot-Remote/main/Install/Install_HotSpotRemote.sh | bash)  on the spotnikv42 seems ok
2) The spotnikv42 do not reboot and I Force to reboot manually.
See screenshot below. 
[CR-Install-SSH.pdf](https://github.com/user-attachments/files/18193365/CR-Install-SSH.pdf)
3) The apk cannot find the spotnikv42 on the network

I may have made a mistake or forgotten something. 73. F1GLR

**Comment by f4bpp** [comment]
Hello, Thanks for the feedback. Sorry for the installation script that had an issue. I've fixed all now. Please try the new script.

**Comment by bobstar-fr** [comment]
Hello, Thanks for new script.
The new install script is OK and the  hotspot reboot.  See screenshoot below.
[CR-Install-SSH-OK.pdf](https://github.com/user-attachments/files/18207686/CR-Install-SSH-OK.pdf)
But the apk cannot find the spotnikv42 on the network.
The hotspot is called spotnikv42 and is located in RJ45 on the same network as the spmatphone wifi
73. F1GLR

**Comment by f4bpp** [comment]
Bonjour Robert. Avec un indicatif français, je pesne qu'on va pouvoir poursuivre dans la langue de Molière.
La seule explication du problème d’appairage est que le HotSpot est connecté à un réseau WIFI du domicile mais pas celui de la Box Internet. J'ai eu le même problème avec une imprimante inaccessible au domicile parce qu'elle était connectée au réseau WifI de ma freebox alors que dans le séjour, je tentais d'y accéder via un routeur wifi au lieu d'un répéteur WiFi.

**Comment by bobstar-fr** [comment]
Bonjour David. Ok pour la langue FR.
Le hotspot est en RJ45 et le smartphone en wifi sur le même réseau connecté à internet et issu d'un routeur connecté à la box... j'accède au tableau de bord d'origine (F5NLV) du hotspot sans probleme en wifi avec le smartphone et son navigateur mais pas avec l'APK.. Je ne comprends pas ce que la box peut faire.... sauf peut etre si l'APK fait reference a une IP en 192.168.1.yyy auquel cas ça ne peut pas fonctionner... car je suis en 192.168.50.yyy
ou utilise un port spécifique non ouvert sur le routeur. 
73. F1GLR.  Robert

**Comment by f4bpp** [comment]
Est-ce que tu as bien fait les réglages dans le menu "Paramètres" ? Il faut soit sélectionner "spotnikv42.local" dans la liste déroulante ou bien choisir "Personnalisé..." dans la liste et entrer le hostname qui correspond à ton HotSpot. Si via le smartphone, tu accède bien au HotSpot via https://spotnikv42.local il n'y a aucune raison pour que l'appli n'y accède pas. Tu es la première personne à me remonter ce problème et je pense qu'il y a un soucis avec le Hostname... Ou bien une erreur avec l'installation. Il faut vérifier si dans le dossier root il y a bien le script HotSpotRemote.js puis ouvrir le fichier /etc/rc.local et vérifier qu'il y a bien cette ligne :

nohup node /root/HotSpotRemote.js &

**Comment by bobstar-fr** [comment]
Oui, j'ai bien fait les réglages dans le menu "Paramètres", mon hotspot s'appelle bien **spotnikv42** (voir copie écran du routeur ci-contre). [Reseau.pdf](https://github.com/user-attachments/files/18211673/Reseau.pdf)
Je ne peux pas accéder au hotspot (en RJ45) avec le smartphone (en WIFI) via le navigateur et l'adresse [https://spotnikv42.local](https://spotnikv42.local/).
j'ai une erreur : **adresse introuvable** alors que via https://192.168.50.195 ça fonctionne... J'utilise Firefox mobile sur le smartphone Samsung... 
j'ai testé avec "Internet" (le navigateur par défaut du Samsung) et j'ai une précision supplémentaire : "L'adresse DNS de spotnikv42.local est introuvable", et bien sur ça ne fonctionne pas, alors qu'avec https://192.168.50.195 c'est OK comme avec Firefox mobile.
Vu que c'est l'adresse **https://spotnikv42.local** qui semble poser problème, je pense que quelque chose n'est pas passé lors du script d'installation....
J'ai bien vérifié " il y a bien le script HotSpotRemote.js puis ouvrir le fichier /etc/rc.local et vérifier qu'il y a bien cette ligne :
nohup node /root/HotSpotRemote.js &_" .... Tout y est...
**Pour info : Avec l'ordinateur (en RJ45) et sur le même réseau, j’accède bien au tableau de bord du hotspot via [https://spotnikv42.local](https://spotnikv42.local/) mais en créant une exception car c'est détecté comme site non sécurisé
Avec un analyseur de réseau, Il s'agirait d'un problème de signature ou d'auto signature de certificat qui bloque sur le smartphone avec spotnikv42.local en wifi.**
73. F1GLR. Robert

**Comment by bobstar-fr** [comment]
Bonjour
J'ai ressorti une vieille version 3.1.1 sur µSD du spotnik, je l'ai mise en place dans le hotspot et j'ai fait l'installation comme indiqué.... Même résultat... tout y est après vérification en ssh et le hotspot n'est pas détecté par l'APK... 
73.F1GLR. Robert
